x/staking/client/rest/grpc_query_test.go

```diff
@@ -410,7 +409,7 @@ func (s *IntegrationTestSuite) TestQueryDelegationsResponseCode() {
 	val := s.network.Validators[0]
 
 	// Create new account in the keyring.
-	info, _, err := val.ClientCtx.Keyring.NewMnemonic("test", keyring.English, sdk.FullFundraiserPath, keyring.DefaultBIP39Passphrase, hd.Secp256k1)
+	info, _, err := val.ClientCtx.Keyring.NewMnemonic("test", keyring.English, sdk.FullFundraiserPath, hd.Secp256k1)
```

Comment:
@alessio can you clarify what this is doing, and why this is necessary here?

Reply:
API change introduced in the keyring. The build was broken since https://github.com/cosmos/cosmos-sdk/pull/9082/commits/5aec5e6f514b02d19aa9c81533bb7fbbf2e341a9 was merged in

Reply:
@alessio the commit you link here is the one from this PR. Is there another commit with the actual prior change? I'd like to understand if we need to have this fixed on master, or if not, why this has diverged?

Reply:
Answering my own question here. The new keyring api [on master](https://github.com/cosmos/cosmos-sdk/blob/master/x/staking/client/rest/grpc_query_test.go#L421) has not actually been backported to v0.42.x yet, so this needs to diverge from master until we backport the keyring API to v0.42.x